app/views/users/show.html.erb

```diff
@@ -1,20 +1,22 @@
 <%= content_for(:title) { "#{confirmed_name(@user)} \\\\ #{t('users.index.users')}" } %>
 
 <h2><%= t(".user") -%></h2>
+<hr/>
+<div class="well">
+  <dl class="dl-horizontal">
+    <dt style="width: 200px; margin-right: 10px"><%= t('id') %></dt>
```

Comment:
Seems like we should probably make `width: 200px; margin-right: 10px` a class, or maybe you're setting the stage for utility CSS here?

Reply:
re-up on this comment regarding the inline style